web/wp-cli.yml

```diff
@@ -0,0 +1 @@
+path: wp
```

Comment:
This path needs to align with the folder structure defined in this repo, so it makes a lot of sense to commit it here rather than scaffold it from another project.

Reply:
Hi, I am wondering, can you tell me why this file was never removed? There is already one that exists mapped to web/wp in the main directory? Bedrock does not have it either. Do you know why you left it in? And if you do not need it, can it be removed?